src/instructlab/model/list.py

```diff
@@ -0,0 +1,110 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=redefined-builtin
+
+# Standard
+from pathlib import Path
+import logging
+import os
+import time
+
+# Third Party
+from huggingface_hub import ModelCard
+from prettytable import ALL, PrettyTable
+from requests import exceptions as requests_exceptions
+import click
+import pandas
+
+# First Party
+from instructlab.configuration import DEFAULTS
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option(
+    "--model-dirs",
+    help="Base directories where models are stored.",
+    multiple=True,
+    default=[DEFAULTS.MODELS_DIR],
+    show_default=True,
+)
+@click.option(
+    "--list-checkpoints",
+    type=click.BOOL,
+    default=False,
+    is_flag=True,
+    show_default=True,
+)
+def list(model_dirs: str, list_checkpoints: bool):
```

Comment:
The type hint suggest this should be passed as a string, but the default val appears to be a list?